pkg/confgen/grafana_jsonnet.go

```diff
@@ -165,32 +168,39 @@ type Dashboard struct {
 	Panels []byte
 }
 
-func (cg *ConfGen) generateGrafanaJsonnet(folderName string) error {
+func (dashboard *Dashboard) generateDashboardJson() []byte {
```

Comment:
Please consider renaming to `generateDashboardJsonnet()` because the output is jsonnet rather than json.